README.md

```diff
@@ -17,7 +17,7 @@ Two main sets of functionalities are provided:
 
 ## `display` command
 
-The CLI command outputs a graphical representation of the graph as a PNG image.
+The plugin is currently developed using version `2.7.0` of renku-python. Please ensure that this version is installed by running:
```

Comment:
```suggestion
The plugin is currently developed using version `2.7.0` of renku-python. Please ensure that a  compatible version (starting with `2` and at least `2.7`) is installed by running:
```

Reply:
mmmm but within the setup we want renku version to be at least `2.7` ?

Reply:
so it is the same as I wrote?